lib/SearchAndSort/tests/interactor.js

```diff
@@ -19,6 +19,8 @@ export default interactor(class SearchAndSortInteractor {
   expandFilterPaneButton = new ExpandFilterPaneButtonInteractor();
 
   resultsSubtitle = text('#paneHeaderpane-results-subtitle');
+  customPaneSub = text('[data-test-custom-pane-sub]');
```

Comment:
```suggestion
  customPaneSub = scoped('[data-test-custom-pane-sub]');
```
and then use it for retrieving text and information about whether the component is present.